server/tests/api/videos/video-imports.ts

```diff
@@ -156,6 +157,11 @@ C'est un sous-titre français
 
 00:00:10.000 --> 00:00:14.000
 Ajouter un sous-titre est vraiment facile`)
+
+      expect(res.body.video.thumbnailPath).to.equal(`/static/thumbnails/${res.body.video.uuid}.jpg`)
```

Comment:
Please add also this check line 122, because we should have the thumbnail/preview in the import response